nvme_gw_server.py

```diff
@@ -277,6 +287,17 @@ def nvmf_get_subsystems(self, request, context):
 
         return pb2.subsystems_info(subsystems=json.dumps(ret))
 
+    def ping(self):
+        try:
+            ret = self.spdk_rpc.spdk_get_version(
+                self.client,
+            )
```

Comment:
Nit: this would fit on a single line.  Also, no need to assign to ret.

Reply:
done